Traceback (most recent call last):
  File "C:\Users\alber\anaconda3\envs\ambiente_1\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\alber\anaconda3\envs\ambiente_1\Lib\site-packages\nbclient\client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\alber\anaconda3\envs\ambiente_1\Lib\site-packages\jupyter_core\utils\__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\alber\anaconda3\envs\ambiente_1\Lib\asyncio\base_events.py", line 686, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\alber\anaconda3\envs\ambiente_1\Lib\site-packages\nbclient\client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "C:\Users\alber\anaconda3\envs\ambiente_1\Lib\site-packages\nbclient\client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\alber\anaconda3\envs\ambiente_1\Lib\site-packages\nbclient\client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
**Explicaci√≥n**:aüîπ Q1 (Primer cuartil)
Es el borde inferior de la caja azul.

Representa el valor por debajo del cual est√° el 25% de los d√≠as.

En la imagen, Q1 parece estar en 2 ‚Üí Esto significa que un cuarto de los d√≠as tuvo 2 defectos o menos.

üî∏ Q2 (Mediana)
Es la l√≠nea roja dentro de la caja.

Marca el valor central: el 50% de los datos est√°n por debajo, y el 50% por encima.

En el gr√°fico, la mediana est√° en 3 ‚Üí Es el n√∫mero de defectos t√≠pico de un d√≠a.

üîπ Q3 (Tercer cuartil)
Es el borde superior de la caja.

Representa el punto por debajo del cual est√° el 75% de los datos.

Aqu√≠ parece estar en 4 ‚Üí Es decir, el 75% de los d√≠as tuvo 4 defectos o menos.

üß† ¬øQu√© aprendemos?
La mayor√≠a de los d√≠as tienen entre 2 y 4 defectos (la "caja").

D√≠as con m√°s de 4 ya est√°n en el 25% m√°s alto.

Algunos puntos m√°s arriba (como el de 8) son valores inusuales, y podr√≠an indicar fallos importantes.
------------------


[1;36m  Cell [1;32mIn[6], line 1[1;36m[0m
[1;33m    **Explicaci√≥n**:aüîπ Q1 (Primer cuartil)[0m
[1;37m                     ^[0m
[1;31mSyntaxError[0m[1;31m:[0m invalid character 'üîπ' (U+1F539)


